cmd/types.go

```diff
@@ -184,7 +184,14 @@ type noopSubmitter struct {
 	log     *log.Logger
 }
 
-var _ resultSubmitter = &noopSubmitter{}
+func newNoopSubmitter(emitLog bool, log *log.Logger) *noopSubmitter {
+	return &noopSubmitter{
+		emitLog: emitLog,
+		log:     log,
```

Comment:
This gets past the linter issue, but I think we'd want to keep the `message` as an option, but I'll let @bcrochet or @komish also chime in.

Reply:
The `reason` field is still available in the struct.  It's just default to an empty string.